pvlib/iotools/psm3.py

```diff
@@ -20,12 +21,31 @@
     'surface_pressure', 'wind_direction', 'wind_speed')
 PVLIB_PYTHON = 'pvlib python'
 
+# Dictionary mapping PSM3 names to pvlib names
+VARIABLE_MAP = {
+    'GHI': 'ghi',
+    'DHI': 'dhi',
+    'DNI': 'dni',
+    'Clearsky GHI': 'ghi_clear',
+    'Clearsky DHI': 'dhi_clear',
+    'Clearsky DNI': 'dni_clear',
+    'Solar Zenith Angle': 'apparent_zenith',
```

Comment:
Oh, maybe we should hold off merging for a moment -- https://github.com/pvlib/pvlib-python/pull/717#issuecomment-1043365592

Reply:
Since a decision has been made, I have renamed `apparent_zenith` to `solar_zenith`.